winpr/libwinpr/file/generic.c

```diff
@@ -785,11 +785,54 @@ struct _WIN32_FILE_SEARCH
 };
 typedef struct _WIN32_FILE_SEARCH WIN32_FILE_SEARCH;
 
+static BOOL FindDataFromStat(const char* path, const struct stat* fileStat,
+                             LPWIN32_FIND_DATAA lpFindFileData)
+{
+	UINT64 ft;
+	char* lastSep;
+	lpFindFileData->dwFileAttributes = 0;
```

Comment:
Since its an api, we need to check the pointer parameters at least for NULL

if (!lpFindFileData)
 		return FALSE;

Reply:
Internal function, already checked